app/services/ChargeHistoryService.scala

```diff
@@ -47,12 +47,8 @@ class ChargeHistoryService @Inject()(chargeHistoryConnector: ChargeHistoryConnec
         AdjustmentHistoryModel(creation, List.empty)
       case _ =>
         val creation = AdjustmentModel(amount = chargeHistory.minBy(_.documentDate).totalAmount, adjustmentDate = None, reasonCode = "create")
-        val poaAdjustmentHistory: List[AdjustmentModel] = adjustments(chargeHistory.filter(_.poaAdjustmentReason.isDefined).sortBy(_.reversalDate), documentDetail.originalAmount)
-        val otherAdjustmentHistory: List[AdjustmentModel] = chargeHistory.filter(_.poaAdjustmentReason.isEmpty).sortBy(_.reversalDate).map(
-          event => AdjustmentModel(event.totalAmount, Some(event.reversalDate), event.reasonCode)
-        )
-        val fullAdjustmentHistory: List[AdjustmentModel] = poaAdjustmentHistory ++ otherAdjustmentHistory
-        AdjustmentHistoryModel(creation, fullAdjustmentHistory.sortBy(_.adjustmentDate))
+        val poaAdjustmentHistory: List[AdjustmentModel] = adjustments(chargeHistory.sortBy(_.reversalDate), documentDetail.originalAmount)
+        AdjustmentHistoryModel(creation, poaAdjustmentHistory.sortBy(_.adjustmentDate))
```

Comment:
@danielmaltz1  - do we have relevant test to cover this change? or this is just a refactoring?